src/saturn_engine/worker/executors/queue.py

```diff
@@ -53,13 +55,25 @@ async def run_queue(self) -> None:
                 async with processable._context:
 
                     @self.services.s.hooks.message_executed.emit
-                    async def scope(xmsg: ExecutableMessage) -> PipelineResults:
-                        return await self.executor.process_message(xmsg.message)
+                    async def scope(
+                        xmsg: ExecutableMessage,
+                    ) -> PipelineResults:
+                        try:
+                            return await self.executor.process_message(xmsg.message)
+                        except Exception as err:
+                            self.logger.exception(err)
```

Comment:
rm?

Reply:
I removed the exception logging, was there anything else here?